routing/routing_tests/turns_tts_text_tests.cpp

```diff
@@ -156,6 +156,77 @@ UNIT_TEST(GetTtsTextTest)
   TEST_EQUAL(getTtsText.GetTurnNotification(notification4), "Затем. Поворот налево.", ());
 }
 
+UNIT_TEST(GetTtsStreetTextTest)
+{
+  string const engShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"In 300 meters.\",\
+      \"in_500_meters\":\"In 500 meters.\",\
+      \"then\":\"Then.\",\
+      \"make_a_right_turn\":\"Make a right turn.\",\
+      \"make_a_left_turn\":\"Make a left turn.\",\
+      \"dist_direction_onto_street\":\"%1$s %2$s onto %3$s\",\
+      \"you_have_reached_the_destination\":\"You have reached the destination.\"\
+      }";
+
+  string const jaShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"三百メートル先\",\
+      \"in_500_meters\":\"五百メートル先\",\
+      \"then\":\"その先\",\
+      \"make_a_right_turn\":\"右折です。\",\
+      \"make_a_left_turn\":\"左折です。\",\
+      \"make_a_right_turn_street\":\"右折し\",\
+      \"make_a_left_turn_street\":\"左折し\",\
+      \"dist_direction_onto_street\":\"%1$s%2$s %3$s に入ります\",\
+      \"you_have_reached_the_destination\":\"到着。\"\
+      }";
+
+  string const faShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"ﺩﺭ ﺲﯿﺻﺩ ﻢﺗﺮﯾ\",\
+      \"in_500_meters\":\"ﺩﺭ ﭖﺎﻨﺻﺩ ﻢﺗﺮﯾ\",\
+      \"then\":\"ﺲﭙﺳ\",\
+      \"make_a_right_turn\":\"ﺐﻫ ﺭﺎﺴﺗ ﺐﭙﯿﭽﯾﺩ.\",\
+      \"make_a_left_turn\":\"ﺐﻫ ﭻﭘ ﺐﭙﯿﭽﯾﺩ.\",\
+      \"dist_direction_onto_street\":\"%1$s %2$s ﺐﻫ %3$s\",\
+      \"you_have_reached_the_destination\":\"ﺶﻣﺍ ﺮﺴﯾﺪﻫ ﺎﯾﺩ.\"\
+      }";
+
+  GetTtsText getTtsText;
+  // Notification(uint32_t distanceUnits, uint8_t exitNum, bool useThenInsteadOfDistance,
+  //    CarDirection turnDir, Settings::Units lengthUnits, std::string nextStreet)
+  Notification const notification1(500, 0, false, CarDirection::TurnRight,
+                                   measurement_utils::Units::Metric, "Main Street");
+  Notification const notification2(300, 0, false, CarDirection::TurnLeft,
+                                   measurement_utils::Units::Metric, "Main Street");
+  Notification const notification3(300, 0, false, CarDirection::TurnLeft,
+                                   measurement_utils::Units::Metric);
+  Notification const notification4(0, 0, true, CarDirection::TurnLeft,
+                                   measurement_utils::Units::Metric);
+
+  getTtsText.ForTestingSetLocaleWithJson(engShortJson, "en");
+  TEST_EQUAL(getTtsText.GetTurnNotification(notification1), "In 500 meters Make a right turn onto Main Street", ());
+  TEST_EQUAL(getTtsText.GetTurnNotification(notification2), "In 300 meters Make a left turn onto Main Street", ());
+  TEST_EQUAL(getTtsText.GetTurnNotification(notification3), "In 300 meters. Make a left turn.", ());
+  TEST_EQUAL(getTtsText.GetTurnNotification(notification4), "Then. Make a left turn.", ());
```

Comment:
If there are some points in sentences to make them sound better, it should be mentioned in a comment in data/strings/sound.txt

Reply:
What kind of points are you thinking of? This is spoken just fine in English.